<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeAddressDialog</class>
 <widget class="QDialog" name="ChangeAddressDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="bb_main">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_specific">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change address of a specific module</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_readdress_mode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>311</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set/change address of a module in readress mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Original module address:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_origin_addr">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>New address:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_new_address">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bb_main</sender>
   <signal>accepted()</signal>
   <receiver>ChangeAddressDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bb_main</sender>
   <signal>rejected()</signal>
   <receiver>ChangeAddressDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
